content/blog/building-warpinator-for-almalinux.md

```diff
@@ -0,0 +1,204 @@
+---
+title: "Building Warpinator for AlmaLinux OS"
+type: blog
+author: 
+ name: "Sofia Boldyreva"
+ bio: "Technical Writer for AlmaLinux OS Project"
+ image: /users/sofia-boldyreva.png
+date: '2023-06-23'
+post:
+    title: "Building Warpinator for AlmaLinux OS"
+    image: /blog-images/warpinator.png
+
+---
+
+## About 
+[Warpinator](https://github.com/linuxmint/warpinator), a file-sharing tool by Linux Mint, exemplifies their commitment to a seamless user experience. We owe its success to the dedicated [contributors](https://github.com/linuxmint/warpinator/graphs/contributors) - thank you!
+
+Originally designed for Debian-based Linux, Warpinator has grown to serve Windows, Android, iOS, and now RPM-based Linux distros, expanding its capabilities to an increasingly diverse user base.
+
+As [I've begun](https://almalinux.org/blog/building-pantheon-for-almalinux-9/) to gain some experience in building rpm packages, I've decided to continue and try to build this useful app for AlmaLinux OS 8 and AlmaLinux OS 9.
```

Comment:
```suggestion

Hi again! As [I've begun](https://almalinux.org/blog/building-pantheon-for-almalinux-9/) to gain some experience in building rpm packages, I've decided to continue and try to build this useful app for AlmaLinux OS 8 and AlmaLinux OS 9.

## About 
[Warpinator](https://github.com/linuxmint/warpinator), a file-sharing tool by Linux Mint, exemplifies their commitment to a seamless user experience. We owe its success to the dedicated [contributors](https://github.com/linuxmint/warpinator/graphs/contributors) - thank you!

Originally designed for Debian-based Linux, Warpinator has grown to serve Windows, Android, iOS, and now RPM-based Linux distros, expanding its capabilities to an increasingly diverse user base.
```

Reply:
I moved your last line to the top, above the 'about' to give readers a little context about what they would learn in this post.